packages/node-build-scripts/package.json

```diff
@@ -35,7 +35,7 @@
         "postcss-discard-comments": "^6.0.0",
         "postcss-scss": "^4.0.9",
         "postcss-simple-vars": "^7.0.1",
-        "prettier": "~2.8.8",
+        "prettier": "^3.0.3",
```

Comment:
prettier is only used here to format the generated Sass & Less variables, and I think the whitespace of those files is not really part of the public API, so I think this is a non-breaking change.